package.json

```diff
@@ -6,6 +6,9 @@
     "email": "angelo@angeloanan.xyz",
     "url": "https://angeloanan.xyz"
   },
+  "engines": {
+    "node": ">=18.0.0"
+  },
```

Comment:
Our build is failing since deployment platform (Vercel) does not support Node 16. 

I think it is fine if we remove this. I understand that we are using native fetch and node/core fetch landed in Node 18, but we are running GH actions on Node 18.